docs/reference/hooks/interchain-gas.mdx

```diff
@@ -113,3 +172,54 @@ If the `handle` call consumes more gas than quoted, the relayer will not submit
 
 </TabItem>
 </Tabs>
+
+## Using the Hyperlane Explorer for Debugging
+
+The Hyperlane Explorer is a powerful tool to debug cross-chain message issues, including gas payments and relayer behavior.
+
+### Key Features
+
+- **Message Status**: View the current status of your message (e.g., "Retry: GasPaymentRequirementNotMet").
+- **Gas Payment Details**: Check the gas amount paid (Origin IGP gas amount) and the amount required by the relayer.
+- **Simulate Calldata**: Use the "View calldata details" option to simulate transactions on tools like Tenderly.
+
+## Troubleshooting
+
+This section covers common issues developers encounter with Interchain Gas Payments, along with potential solutions.
+
+### `GasPaymentRequirementNotMet` Warning
+
+- **Reason:** This occurs when the gas payment provided during `dispatch` does not meet the relayer's calculated requirement.
+
+- **Solution:**
+  - Use `quoteGasPayment` to calculate the required payment for the destination domain and gas limit.
+  - Verify that `msg.value` in your metadata covers the relayer’s quoted fee.
+  - Check the message status in the Hyperlane Explorer. Look for: `Retry(GasPaymentRequirementNotMet)`.
+
+### High Testnet Gas Prices
+
+- **Reason:** Some testnets have limited gas token liquidity, which results in inflated gas prices.
+
+- **Solution:**
+  - Simulate the transaction to estimate realistic gas costs.
+  - Be aware that testnet relayers may charge higher fees due to low token availability.
+  - Consider testing on a different network with lower gas token prices, if possible.
```

Comment:
im not sure about this one

Reply:
removed, was in one of tg but seems not necessary/common